File: problem1-9/problem04.py
# -*- coding: utf-8 -*-
"""
Created on Sat Oct 26 21:49:03 2019

@author: samet
"""
def check_palindrome(num):
    n = len(num)-1
    for i in range(0,n):
        if(num[i]!=num[n-i]):
            return False
    return True
max=0
for i in range(999,100,-1):
    for j in range(999,100,-1):
        if((i*j)>max and check_palindrome(str(i*j))):
            max = i*j
print(max)

File: problem1-9/problem04.js
function checkPalindrome(num) {
    return num === num.split('').reverse().join('');
}

let max = 0;

for (let i = 999; i > 100; i--) {
    for (let j = 999; j > 100; j--) {
        let num = i * j;
        if (num > max && checkPalindrome(num.toString())) max = num;
    }
}

console.log(max);